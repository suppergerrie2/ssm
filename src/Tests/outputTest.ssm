ldc 90
ldc 10
ldc 71
ldc 70
ldc 69
ldc 68
ldc 67
ldc 66
ldc 65
trap 1
trap 1
trap 1
trap 1
trap 1
trap 1
trap 1
trap 1
trap 0

LDC 12
trap 0

LDC 451
trap 0

ldc 71
ldc 70
ldc 69
ldc 68
ldc 67
ldc 66
ldc 65
trap 1
trap 1
trap 1
trap 1
trap 1
trap 1
trap 1