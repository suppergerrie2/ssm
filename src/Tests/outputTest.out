ABCDEFG
90
12
451
ABCDEFG